src/scss/themes/_hero.scss

```diff
@@ -180,3 +180,54 @@
 		}
 	}
 }
+
+// Hero Package theme — extra
+@mixin oTeaserHeroExtra {
+	#{$o-hoverable-if-hover-enabled} .o-teaser__heading:hover,
+	.o-teaser__heading:focus {
+		color: $_o-teaser-color-extra-highlights;
+	}
+	#{$o-hoverable-if-hover-enabled} .o-teaser__tag:hover,
+	.o-teaser__tag:focus {
+		color: oTeaserTextColor(#ffffff, $_o-teaser-color-extra-highlights, 73);
+	}
+
+	.o-teaser__meta {
+		color: $_o-teaser-color-extra-highlights;
+	 	&::after {
+			border-color: $_o-teaser-color-extra-highlights;
+		}
+	}
+
+	.o-teaser__heading {
+		color: #ffffff;
+	}
+
+	.o-teaser__standfirst {
+		background-color: oTeaserTextColor(#ffffff, $_o-teaser-color-extra-background, 80);
+	}
+
+	&.o-teaser--hero {
```

Comment:
Why is this required? Based on the way `oTeaserHeroExtra` is being included this will result in the selector `.o-teaser--extra-article.o-teaser--hero.o-teaser--hero`, is that the intended effect?